### PR TITLE
Optional args

### LINKED ISSUE #28: Document optional args
https://github.com/target/flottbot/pull/86

### PR DESCRIPTION
**Pull request by wass3r**
Closes #28 

- Document new feature added via https://github.com/target/flottbot/pull/86
- Gutenberg changed to Zola. This contains the upgrade to that.

### CODE REVIEW

**Comment by wass3r** [comment]
Should wait to merge until https://github.com/target/flottbot is tagged.

**Comment by JordanSussman** [comment]
I think the documentation needs to be updated to clearly state that when referencing optional args you will strip the `?`.

**Comment by wass3r** [comment]
Yea, I did it in the example but not in the copy. I reworded it now - open to suggestions.

File: content/rules/anatomy.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ active: true # activate rule
 respond: echo # text match to trigger this rule to take action, bot is looking for the word "echo".
 args: # this is an array of args to pass in after a match is found
   - message # first string capture after the match will be mapped to the variable ${message}
+  - arg? # this second argument is optional and will be availabe as ${arg}, ie. the rule will process even if this argument is missing.
```
Comment (JordanSussman) [inline content/rules/anatomy.md]: availabe > available

**Comment by JordanSussman** [comment]
Everything LGTM besides the one minor typo.

File: content/rules/anatomy.md (hunk 1 above)
Comment (wass3r) [reply content/rules/anatomy.md]: Thanks - fixed

**Comment by wass3r** [comment]
We'll wait to merge until main repo gets tagged as mentioned previously

**Comment by wass3r** [comment]
available in 0.2.0 (https://github.com/target/flottbot/releases/tag/0.2.0)

**Pull request merged**